evennia/utils/spawner.py

```diff
@@ -153,8 +154,8 @@ def _batch_create_object(*objparams):
     # bulk create all objects in one go
 
     # unfortunately this doesn't work since bulk_create doesn't creates pks;
-    # the result are double objects at the next stage
-    #dbobjs = _ObjectDB.objects.bulk_create(dbobjs)
+    # the result are database objects at the next stage
```

Comment:
This should really be "double" - it's a possible bug that is being documented. I'll edit it back when I merge.